### DISCUSSION TITLE
Performance

### DISCUSSION THREAD

**Discussion by rserafim** [prisma/prisma#11780] [Q&A] [answered]
Do you have any material that shows the response times performance of Prisma vs others?

**Comment by nurul3101** [comment, marked as answer]
Hey @rserafim 👋🏻 ,
Here is a guide on comparing [Prisma vs TypeORM vs Sequalize](https://www.prisma.io/docs/concepts/more/comparisons).
And here is an extensive summary of [Comparing ORMs, Query Builders and Database Libraries](https://www.prisma.io/dataguide/database-tools/top-nodejs-orms-query-builders-and-database-libraries).
This should help you gauge all the possibilities of ORMs which you could use in aspect of Popularity, Maturity, Stability etc.

Please let me know if you need any more information.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
